src/types.ts

```diff
@@ -3,46 +3,40 @@ import * as ethers from "ethers";
 export type Maybe<T> = T | undefined | null;
 
 /**
- * Configuration Object
+ * Reusable Sale Config 
  */
-export interface ClaimSaleConfig {
-  /**
-   * Address of the sale contract
-   */
-  contractAddress: string;
-
-  /**
-   * web3 provider to access blockchain with (on read operations)
-   */
-  web3Provider: ethers.providers.Provider;
-
-  /**
-   * Registrar that holds the domains being used to claim for this sale.
-   */
-  claimingRegistrarAddress: string;
-
-  /**
-   * Advanced settings / properties
-   */
-  advanced?: {
-    /**
-     * IPFS Gateway to use
-     * (Should be fully formed, ie: https://ipfs.fleek.co/ipfs)
-     */
-    ipfsGateway?: string;
-  };
-}
+export type SaleConfiguration = {
+  sellerWallet: string;
+  parentDomainId: string;
+  publicSalePrice: string;
+  privateSalePrice: string;
+  mintlistSaleDuration: number;
+  amountForSale: number;
+  mintlistMerkleRoot: string;
+  startingMetadataIndex: number;
+  folderGroupID: number;
+  publicSaleLimit: number;
+};
+
+export type SaleData = {
+  saleId: number;
+  salePhaseName: string;
+  saleStartTime: string;
```

Comment:
is this `saleStartTime` a string? I saw it was number in the above code

Reply:
fixed